### DISCUSSION TITLE
Way to exclude certain methods from rest impl

### DISCUSSION THREAD

**Discussion by ParidaGopal** [canton7/RestEase#237] [Q&A]
Use case:   I have an interface which has 2 methods corresponding to 2 [GET] operations, 
but now I want to have one more method, but that method should not be a part of any rest call, rather it can be called on the client

```
ex:- 
interface IMyApi
{
  [Get("/book")]
  GetBook();

  AddOauthToken(); // should not be part of rest call
}
 var client = RestClient.For<IMyApi>(someurl);

 use 1 :   client.GetBook()
 use 2 :  client.AddOthToken();
```

Can we have some `[ExcludeRest]` attrib which can be used to decorate methods when we want it to be excluded from rest impl ?

**Comment by canton7** [comment]
I don't think I follow. RestEase works by generating a class which implements your interface. If RestEase were to avoid implementing a method on the interface, that would cause a compiler error, because you can't implement only some of the methods on an interface.

In your example, who provides the implementation for `AddOauthToken`?
